src/Helpers.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Helpers.hpp"
+
+using namespace radar_base;
+
+void helpers::updateEchoes(Radar const& radar_echo,
+    base::Angle const& yaw_correction,
+    std::vector<uint8_t>& world_echoes)
+{
+    double angle = (radar_echo.start_heading).getRad() + yaw_correction.getRad();
+    if (angle < 0) {
+        angle = 2 * M_PI + angle;
+    }
+
+    int start_angle_unit = round(angle / abs(radar_echo.step_angle.getRad()));
+    int angles_in_a_frame = round(2 * M_PI / abs(radar_echo.step_angle.getRad()));
+    int signal = abs(radar_echo.step_angle.getRad()) / radar_echo.step_angle.getRad();
```

Comment:
No need to change this, but the same can be done with `copysign(1, radar_echo.step_angle.getRad())`

Reply:
Actually, isnt the step_angle always between 0-8191? How can it be negative?

Reply:
step_angle is the actual angle in radians, it can be negative.

Reply:
Ah yes, it is a base::Angle, somehow that got past me.